scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -428,7 +428,7 @@ private Widget prepareWorlflowInformationWidget() {
         }
 
         if (documentation != null) {
-            widgetDocumentation = "<b>Documentation :</b> <a href=" + documentation + " target=_blank>" +
+            widgetDocumentation = "<b>Documentation :</b> <a href=/doc/" + documentation + " target=_blank>" +
```

Comment:
@mboussaa This is an absolute Path. it will open https://doc/{{documentation}}....
You need to use a relative path : doc/....